specs/_features/eip7732/fork-choice.md

```diff
@@ -195,27 +187,37 @@ def notify_ptc_messages(
             )
 ```
 
-### `is_payload_present`
+### `is_payload_timely`
 
 ```python
-def is_payload_present(store: Store, beacon_block_root: Root) -> bool:
+def is_payload_timely(store: Store, beacon_block_root: Root) -> bool:
     """
-    Return whether the execution payload for the beacon block with root ``beacon_block_root`` was voted as present
-    by the PTC
+    Return whether the execution payload for the beacon block with root ``beacon_block_root``
+    was voted as present by the PTC, and was locally determined to be available
     """
     # The beacon block root must be known
     assert beacon_block_root in store.ptc_vote
-    return store.ptc_vote[beacon_block_root].count(True) > PAYLOAD_TIMELY_THRESHOLD
+    return (
+        sum(store.ptc_vote[beacon_block_root]) > PAYLOAD_TIMELY_THRESHOLD
```

Comment:
Another note more than a comment but I believe with a "signalling" approach (as opposed to block/slot voting) as with this forkchoice, it may be better to have the `ptc_vote` to have three statuses: no vote, present, absent. Since anyway we will need to modify this map when rebasing on top of Fulu to keep separate track of payload availability and data availability, it may be good to set this type right from the start.